README.md

```diff
@@ -0,0 +1,24 @@
+#Modified pyinstaller fork
```

Comment:
Why are we not modifying the original .rst file and instead making our own and deleting the prior one?

Reply:
And why is modifying the original .rst instead of making our own and delting the prior one better?

Reply:
Contains far more detail on how to build and setup.